certified-connectors/Asite (KSA)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1163 @@
+              "value-path": "folderId",
+              "parameters": {
+                "projectId": {
+                  "parameter": "projectId"
+                },
+                "fileFilter": [],
+                "hasSubFolder": true
+              }
+            },
+            "x-ms-dynamic-tree": {
+              "settings": {
+                "canSelectParentNodes": true,
+                "canSelectLeafNodes": false
+              },
+              "open": {
+                "operationId": "GET_ASITE_FOLDER_LIST",
+                "itemValuePath": "folderId",
+                "itemTitlePath": "folder_title",
+                "itemFullTitlePath": "folderPath",
+                "itemIsParent": "true",
+                "itemsPath": "folderData",
+                "parameters": {
+                  "projectId": {
+                    "parameterReference": "projectId"
+                  },
+                  "isFolderDataReq": {
+                    "value": true
+                  },
+                  "isWorkspace": {
+                    "value": 1
+                  },
+                  "Accept": {
+                    "value": "*/*"
+                  }
+                }
+              },
+              "browse": {
+                "operationId": "GET_ASITE_SUB_FOLDER_LIST",
+                "itemValuePath": "folderId",
+                "itemTitlePath": "folder_title",
+                "itemFullTitlePath": "folderPath",
+                "itemIsParent": "true",
+                "parameters": {
+                  "projectId": {
+                    "parameterReference": "projectId"
+                  },
+                  "folderId": {
+                    "selectedItemValuePath": "folderId"
+                  },
+                  "hasSubFolder": {
+                    "value": true
+                  }
+                }
+              }
+            }
+          },
+          {
+            "name": "fileName",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "File Name",
+            "description": "Enter File name with file type extension"
+          },
+          {
+            "name": "metadataId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "File Metadata",
+            "description": "Select Metadata ID from Dynamic Content section"
+          },
+          {
+            "name": "fileBinary",
+            "in": "body",
+            "x-ms-summary": "File Content",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "format": "binary",
+              "description": "Binary file content"
+            }
+          }
+        ]
+      }
+    },
+    "/getFolderAttributesFromTrigger": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK, Success",
+            "schema": {}
```

Comment:
@asitedeveloper Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.** 

Applies the same comment all the empty schemas in the PR

Reply:
@sriyen-msft We have sent the dynamic schema in response to our API call. Hence we have set schema as empty in swagger JSON. The dynamic schema that we are sending in API response is as below. The fields in this schema is fully dynamic and based on values selected for an API call. So it can be different for different values of the respective parameters. So as it is not fixed, we have kept the schema as empty.

{
	"type": "object",
	"properties": {
		"DocStatus": {
			"type": "string",
			"title": "DocStatus"
		},
		"PublisherOrg": {
			"type": "string",
			"title": "PublisherOrg"
		}
	}
}